docs/clients/coil_temperature_estimator_full.rst

```diff
@@ -0,0 +1,158 @@
+Coil Temperature Estimator
+--------------------------
+
+The Coil Temperature Estimator is a convection and conduction based thermal model to estimate the temperature
+of motor coils when they are not directly sensed.  The temperature is used to derate the motor if the 
+temperature rises into dangerous levels. The temperature limits can be adjusted, though this is not recommended.
+The convection coefficient should be tuned for the specific propeller used on the motor.  A convection 
+coefficient for an average sized propeller for the given motor is loaded by default.  Consider changing the
+convection coefficient if a relatively large or small propeller is used or if extreme performance, and thus
```

Comment:
Consider changing the convection coefficient if using a relatively large or small propeller, or if extreme performance and accuracy are required.